exercise_01/index.js

```diff
@@ -1,9 +1,31 @@
+import onError from "./callbacks/onError.js";
+import onSuccess from "./callbacks/onSuccess.js";
+import transformStringToArray from "./transformStringToCollection.js";
 //test with value as number and log result
```

Comment:
Improve Code Readability with Clearer Code Blocks. Fix it!
```suggestion

//test with value as number and log result
```